scripts/install/nltk_dictionaries.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eo pipefail
+
+echo "MACHINE LEARNING WORKER NOTE: Next, you will be asked to install some wordlists for the natural language processing parts of Augur.  Please be advised that the machine learning workers, including the clustering worker, discourse worker, messsage_analysis_worker, pull_request_analysis_worker, and the LSTM model of the insight_worker will not function fully without these lists."
+
+read -r -p "Would you like to install required NLTK word lists for machine learning workers? [Y/n] " response
+case "$response" in
+  [yY][eE][sS]|[yY])
+    echo "Installing..."
+    python -m nltk.downloader stopwords 
+    python -m nltk.downloader punkt                
+    python -m nltk.downloader popular 
+    python -m nltk.downloader universal_tagset 
+    echo "Done!"
+    ;;
+  *)
+    echo "Skipping NLTK Word Vector dependencies. Please be advised that the machine learning workers, including the clustering worker, discourse worker, messsage_analysis_worker, pull_request_analysis_worker, and the LSTM model of the insight_worker will not function fully without these lists."
+    ;;
+
+esac
+
+
+echo "**********************************"
+echo "***** NLTK DICTIONARY INSTALLATION  COMPLETE *****"
+echo "**********************************"
```

Comment:
This code is also present in PR #1049. I don't know which version is correct.

Reply:
They are the same.  I've merged that commit in that other branch as well.

Reply:
This is not resolved either. Please fix the formatting as outlined in [this comment](https://github.com/chaoss/augur/pull/1049/files?file-filters%5B%5D=.sh#r544597459) on PR #1049.

Reply:
extra space corrected.